pallet/xrpl-bridge/src/lib.rs

```diff
@@ -105,6 +105,8 @@ pub mod pallet {
 		/// Threshold to emit event TicketSequenceThresholdReached
 		type TicketSequenceThreshold: Get<Percent>;
 
+		type SourceTag: Get<u32>;
```

Comment:
Hope this is going to be a constant.
```suggestion
                #[pallet::constant]
		type SourceTag: Get<u32>;
```

Reply:
Made it flexible... can be set via extrinsic